packages/cojson-storage/src/managerAsync.ts

```diff
@@ -157,54 +147,56 @@ export class StorageManagerAsync {
 
         collectNewTxs({
           newTxsInSession,
-          newContentMessages,
+          contentMessage,
           sessionRow,
           firstNewTxIdx: idx,
           signature: signature.signature,
         });
 
         idx = signature.idx + 1;
+
+        if (signatures.length > 1) {
+          await this.sendContentMessage(coValueRow, contentMessage);
```

Comment:
Now going to send the content messages as they are loaded.

This way we don't block the main thread use the fetching time between two content messages to process and validate it on the LocalNode.